pyproject.toml

```diff
@@ -53,9 +57,29 @@ source = "vcs"
 version-file = "src/GOSTurban/_version.py"
 
 [tool.codespell]
-skip = 'docs/_build,docs/references.bib,__pycache__,*.png,*.gz,*.whl'
+skip = 'docs/_build,docs/references.bib,__pycache__,*.png,*.gz,*.whl,pyproject.toml,*csv,notebooks/Implementations/WSF_DECAT_B_ExploringIDCerrors.ipynb,notebooks/Implementations/URB_SEAU1_NovelUrbanization/MAP_Urbanization.ipynb'
 ignore-regex = '^\s*"image\/png":\s.*'
-ignore-words-list = "gost,"
+ignore-words-list = "gost,GOST,ALOS,ans,NAM"
```

Comment:
This `ignore-words-list` parameter doesn't seem to be working as expected so I had to add the relevant notebooks to the `skip` field above.